app/controllers/user_passwords_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
 
     if resource.errors.empty?
       set_flash_message(:success, :send_instructions) if is_navigational_format?
-      respond_with resource, location: spree.login_path
```

Comment:
I tried to understand where this login_path takes us now and where the modal is loaded but I couldnt find it...

Reply:
`login_path` hits `get "/login", to: redirect("/#/login")` and in turn `/#/login` gets handled by the HomeController. Finally, the `#/login` is what the frontend checks to load the modal. I haven't checked where that final step happens but it's a good-old frontend route.